manifests/host.pp

```diff
@@ -3,12 +3,15 @@
 define dhcp::host (
   Stdlib::IP::Address $ip,
   Dhcp::Mac $mac,
-  String $ddns_hostname                 = $name,
-  Hash $options                         = {},
-  String $comment                       = '',
-  Boolean $ignored                      = false,
-  Optional[Integer] $default_lease_time = undef,
-  Optional[Integer] $max_lease_time     = undef,
+  String $ddns_hostname                  = $name,
+  Hash $options                          = {},
+  String $comment                        = '',
+  Boolean $ignored                       = false,
+  Optional[Integer] $default_lease_time  = undef,
+  Optional[Integer] $max_lease_time      = undef,
+  Optional[Array[String[1]]] $on_commit  = undef,
```

Comment:
ah crap, I was a bit blind when I reviewed this the last time. For array we try to default to empty arrays instead of making them optional. From our [review guidelines](https://voxpupuli.org/docs/#reviewing-a-module-pr):
> Are there new params with datatype Hash or Array? If possible, they should default to empty Hash/Array instead of undef. You can also enforce the datastructure like Array[String[1]]

Reply:
It'd be nice if the tests caught this.  I honestly don't see what difference it makes, but a fix is in the works if it makes this project happy.  IMHO, I think `undef` is far more appropriate to mean "not set", but that's me.